bot/discord/commands/staff/premium.js

```diff
@@ -14,7 +14,7 @@ exports.run = async(client, message, args) => {
         client.channels.cache.get('898041841939783732').send('Thanks, <@' + userid + '> for donating $' + parser.format(amount))
     }
 
-    let userid = message.guild.members.cache.get(args[2].match(/[0-9]{18}/).length == 0 ? args[2] : args[2].match(/[0-9]{18}/)[0]);
+    let userid = message.guild.members.cache.get(args[2].match(/[0-9]{18,19}/).length == 0 ? args[2] : args[2].match(/[0-9]{18,19}/)[0]);
```

Comment:
```suggestion
    let userid = message.guild.members.cache.get(args[2].match(/[0-9]{17,19}/).length == 0 ? args[2] : args[2].match(/[0-9]{17,19}/)[0]);
```

Reply:
already merged once broke the entire command

Reply:
Oh